client/packages/common/src/utils/files/FileUtils.ts

```diff
@@ -177,8 +177,8 @@ const getFilename = (type?: string, title?: string) => {
       break;
   }
 
-  const today = Formatter.naiveDate(new Date());
-  const filename = `${title || 'export'}_${today}.${extension}`;
+  const today = Formatter.toIsoString(new Date()); // to match backend datetime
```

Comment:
Not an exact match to backend but date and time matches so.. good enough for now